app/bundles/CoreBundle/Views/Helper/details.html.twig

```diff
@@ -1,6 +1,6 @@
 {% set mauticTemplateVars = mauticTemplateVars is defined ? mauticTemplateVars : [] %}
 {{ customContent('details.top', mauticTemplateVars) }}
-{% if attribute(entity, 'getCategory') is defined %}
+{% if attribute(entity, 'getCategory') is defined and method_exists(entity, 'getCategory') %}
```

Comment:
Unfortunately. Without it, it will throw an exception. Root cause is https://github.com/mautic/mautic/blob/5.x/app/bundles/CoreBundle/Entity/CommonEntity.php#L36-L45 I figured that using `method_exists` is ok since this is what it does in that method.

This is the same issue with "getPublishUp" as well.

I'm open to another solution as I'm not a huge fan of this either.  I believe this is part of the "strict" being set to true. If this was setting was false, it should work as expected. Reference: https://github.com/twigphp/Twig/blob/3.x/src/Extension/CoreExtension.php#L1604-L1613

Reply:
Ah, the *magic*. Let's keep it in then.